src/lib/helpers.ts

```diff
@@ -344,3 +348,105 @@ export function parseResponseAPIVersion(response: Response) {
     return null
   }
 }
+
+// Taken from simplewebauthn
+
+/**
+ * Convert from a Base64URL-encoded string to an Array Buffer. Best used when converting a
+ * credential ID from a JSON string to an ArrayBuffer, like in allowCredentials or
+ * excludeCredentials
+ *
+ * Helper method to compliment `bufferToBase64URLString`
+ */
+export function base64URLStringToBuffer(base64URLString: string): ArrayBuffer {
```

Comment:
Can switch to base64URL-js as well

Reply:
Yes please let's use our library as that will come in handy for asymmetric JWTs too.

Reply:
Let's please switch to this: https://github.com/supabase-community/base64url-js/blob/main/src/base64url.ts

The README has an example of using `Uint8Array` to convert to Base64-URL and v.v.